CLI/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 6.0
+// swift-tools-version: 6.1
```

Comment:
development in this lib should be in Swift 6.1. We just need to support building in Swift 6.